client/blocks/disconnect-jetpack-dialog/index.jsx

```diff
@@ -0,0 +1,123 @@
+		return { components: { icon: <Gridicon icon={ icon } /> } };
+	}
+
+	planFeatures() {
+		const { plan, translate } = this.props;
+		const features = [];
+		switch ( plan ) {
+			case 'free':
+				features.push(
+					translate( '{{icon/}} Site stats, related content, and sharing tools',
+					this.translateArgs( 'stats-alt' )
+				) );
+				features.push(
+					translate( '{{icon/}} Brute force attack protection and uptime monitoring',
+					this.translateArgs( 'lock' )
+				) );
+				features.push( translate( '{{icon/}} Unlimited, high-speed image hosting', this.translateArgs( 'image' ) ) );
+				break;
+
+			case 'personal':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} Spam filtering', this.translateArgs( 'spam' ) ) );
+				break;
+
+			case 'premium':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push( translate( '{{icon/}} Daily, automated malware scanning', this.translateArgs( 'spam' ) ) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} 13Gb of high-speed video hosting', this.translateArgs( 'video' ) ) );
+				break;
+
+			case 'professional':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push(
+					translate( '{{icon/}} Daily, automated malware scanning with automated resolution',
+					this.translateArgs( 'spam' )
+				) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} Unlimited high-speed video hosting', this.translateArgs( 'video' ) ) );
+				features.push( translate( '{{icon/}} SEO preview tools', this.translateArgs( 'globe' ) ) );
+				break;
+		}
+
+		return features.map( ( freature, index ) => {
+			return (
+				<div key={ 'disconnect-jetpack-dialog__feature-' + index } className="disconnect-jetpack-dialog__feature">
+					{ freature }
+				</div>
+			);
+		} );
+	}
+
+	render() {
+		const { onStay, onDisconnect, isVisible, translate, isBroken, siteName } = this.props;
+		if ( isBroken ) {
+			return (
+				<Dialog isVisible={ isVisible } additionalClassNames="disconnect-jetpack-dialog" onClose={ onStay } >
+				<h1>{ translate( 'Disconnect Jetpack' ) }</h1>
+				<p className="disconnect-jetpack-dialog__highlight">
+					{ translate( 'WordPress.com has not been able to reach example.com for a while.' ) }
+				</p>
+				<div className="disconnect-jetpack-dialog__button-wrap">
+					<Button primary scary onClick={ onDisconnect }>{ translate( 'Remove Site' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Remove Users from Site' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).